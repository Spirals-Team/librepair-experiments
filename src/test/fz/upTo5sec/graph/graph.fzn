array [1..2] of int: X_INTRODUCED_0 = [1,-1];
var 1..8: a:: output_var;
var 1..8: b:: output_var;
var 1..8: c:: output_var;
var 1..8: d:: output_var;
var 1..8: e:: output_var;
var 1..8: f:: output_var;
var 1..8: g:: output_var;
var 1..8: h:: output_var;
var -7..7: X_INTRODUCED_1 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_2 ::var_is_introduced ;
var -7..7: X_INTRODUCED_3 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_4 ::var_is_introduced ;
var -7..7: X_INTRODUCED_5 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_6 ::var_is_introduced ;
var -7..7: X_INTRODUCED_7 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_8 ::var_is_introduced ;
var -7..7: X_INTRODUCED_9 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_10 ::var_is_introduced ;
var -7..7: X_INTRODUCED_11 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_12 ::var_is_introduced ;
var -7..7: X_INTRODUCED_13 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_14 ::var_is_introduced ;
var -7..7: X_INTRODUCED_15 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_16 ::var_is_introduced ;
var -7..7: X_INTRODUCED_17 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_18 ::var_is_introduced ;
var -7..7: X_INTRODUCED_19 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_20 ::var_is_introduced ;
var -7..7: X_INTRODUCED_21 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_22 ::var_is_introduced ;
var -7..7: X_INTRODUCED_23 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_24 ::var_is_introduced ;
var -7..7: X_INTRODUCED_25 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_26 ::var_is_introduced ;
var -7..7: X_INTRODUCED_27 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_28 ::var_is_introduced ;
var -7..7: X_INTRODUCED_29 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_30 ::var_is_introduced ;
var -7..7: X_INTRODUCED_31 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_32 ::var_is_introduced ;
var -7..7: X_INTRODUCED_33 ::var_is_introduced :: is_defined_var;
var 2..7: X_INTRODUCED_34 ::var_is_introduced ;
constraint int_lin_ne(X_INTRODUCED_0,[a,b],0);
constraint int_lin_ne(X_INTRODUCED_0,[a,c],0);
constraint int_lin_ne(X_INTRODUCED_0,[a,d],0);
constraint int_lin_ne(X_INTRODUCED_0,[a,e],0);
constraint int_lin_ne(X_INTRODUCED_0,[a,f],0);
constraint int_lin_ne(X_INTRODUCED_0,[a,g],0);
constraint int_lin_ne(X_INTRODUCED_0,[a,h],0);
constraint int_lin_ne(X_INTRODUCED_0,[b,c],0);
constraint int_lin_ne(X_INTRODUCED_0,[b,d],0);
constraint int_lin_ne(X_INTRODUCED_0,[b,e],0);
constraint int_lin_ne(X_INTRODUCED_0,[b,f],0);
constraint int_lin_ne(X_INTRODUCED_0,[b,g],0);
constraint int_lin_ne(X_INTRODUCED_0,[b,h],0);
constraint int_lin_ne(X_INTRODUCED_0,[c,d],0);
constraint int_lin_ne(X_INTRODUCED_0,[c,e],0);
constraint int_lin_ne(X_INTRODUCED_0,[c,f],0);
constraint int_lin_ne(X_INTRODUCED_0,[c,g],0);
constraint int_lin_ne(X_INTRODUCED_0,[c,h],0);
constraint int_lin_ne(X_INTRODUCED_0,[d,e],0);
constraint int_lin_ne(X_INTRODUCED_0,[d,f],0);
constraint int_lin_ne(X_INTRODUCED_0,[d,g],0);
constraint int_lin_ne(X_INTRODUCED_0,[d,h],0);
constraint int_lin_ne(X_INTRODUCED_0,[e,f],0);
constraint int_lin_ne(X_INTRODUCED_0,[e,g],0);
constraint int_lin_ne(X_INTRODUCED_0,[e,h],0);
constraint int_lin_ne(X_INTRODUCED_0,[f,g],0);
constraint int_lin_ne(X_INTRODUCED_0,[f,h],0);
constraint int_lin_ne(X_INTRODUCED_0,[g,h],0);
constraint int_abs(X_INTRODUCED_1,X_INTRODUCED_2);
constraint int_abs(X_INTRODUCED_3,X_INTRODUCED_4);
constraint int_abs(X_INTRODUCED_5,X_INTRODUCED_6);
constraint int_abs(X_INTRODUCED_7,X_INTRODUCED_8);
constraint int_abs(X_INTRODUCED_9,X_INTRODUCED_10);
constraint int_abs(X_INTRODUCED_11,X_INTRODUCED_12);
constraint int_abs(X_INTRODUCED_13,X_INTRODUCED_14);
constraint int_abs(X_INTRODUCED_15,X_INTRODUCED_16);
constraint int_abs(X_INTRODUCED_17,X_INTRODUCED_18);
constraint int_abs(X_INTRODUCED_19,X_INTRODUCED_20);
constraint int_abs(X_INTRODUCED_21,X_INTRODUCED_22);
constraint int_abs(X_INTRODUCED_23,X_INTRODUCED_24);
constraint int_abs(X_INTRODUCED_25,X_INTRODUCED_26);
constraint int_abs(X_INTRODUCED_27,X_INTRODUCED_28);
constraint int_abs(X_INTRODUCED_29,X_INTRODUCED_30);
constraint int_abs(X_INTRODUCED_31,X_INTRODUCED_32);
constraint int_abs(X_INTRODUCED_33,X_INTRODUCED_34);
constraint int_lin_eq([1,-1,-1],[a,b,X_INTRODUCED_1],0):: defines_var(X_INTRODUCED_1);
constraint int_lin_eq([1,-1,-1],[a,c,X_INTRODUCED_3],0):: defines_var(X_INTRODUCED_3);
constraint int_lin_eq([1,-1,-1],[a,d,X_INTRODUCED_5],0):: defines_var(X_INTRODUCED_5);
constraint int_lin_eq([1,-1,-1],[b,c,X_INTRODUCED_7],0):: defines_var(X_INTRODUCED_7);
constraint int_lin_eq([1,-1,-1],[b,e,X_INTRODUCED_9],0):: defines_var(X_INTRODUCED_9);
constraint int_lin_eq([1,-1,-1],[b,f,X_INTRODUCED_11],0):: defines_var(X_INTRODUCED_11);
constraint int_lin_eq([1,-1,-1],[c,d,X_INTRODUCED_13],0):: defines_var(X_INTRODUCED_13);
constraint int_lin_eq([1,-1,-1],[c,e,X_INTRODUCED_15],0):: defines_var(X_INTRODUCED_15);
constraint int_lin_eq([1,-1,-1],[c,f,X_INTRODUCED_17],0):: defines_var(X_INTRODUCED_17);
constraint int_lin_eq([1,-1,-1],[c,g,X_INTRODUCED_19],0):: defines_var(X_INTRODUCED_19);
constraint int_lin_eq([1,-1,-1],[d,f,X_INTRODUCED_21],0):: defines_var(X_INTRODUCED_21);
constraint int_lin_eq([1,-1,-1],[d,g,X_INTRODUCED_23],0):: defines_var(X_INTRODUCED_23);
constraint int_lin_eq([1,-1,-1],[e,f,X_INTRODUCED_25],0):: defines_var(X_INTRODUCED_25);
constraint int_lin_eq([1,-1,-1],[e,h,X_INTRODUCED_27],0):: defines_var(X_INTRODUCED_27);
constraint int_lin_eq([1,-1,-1],[f,g,X_INTRODUCED_29],0):: defines_var(X_INTRODUCED_29);
constraint int_lin_eq([1,-1,-1],[f,h,X_INTRODUCED_31],0):: defines_var(X_INTRODUCED_31);
constraint int_lin_eq([1,-1,-1],[g,h,X_INTRODUCED_33],0):: defines_var(X_INTRODUCED_33);
solve  satisfy;
