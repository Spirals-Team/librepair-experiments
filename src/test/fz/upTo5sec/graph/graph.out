a = 2;
b = 5;
c = 8;
d = 6;
e = 3;
f = 1;
g = 4;
h = 7;
----------